### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by afuetterer**
I did not create an issue first, because (again) the changes are trivial.

This PR fixes a few minor typos in docstrings and docs.

I used [typos](https://github.com/crate-ci/typos) for this. You can also set up a pre-commit hook, so that all changes run through the spellchecker before commit.

I was unsure about this: typos wanted to change `highligh_color()` to `highlight_color()`, which is modifying the API. That is your decision.
https://github.com/MaartenGr/BERTopic/blob/6f91b4ede448cc40bf6651a8caa25b4f8973651c/bertopic/plotting/_approximate_distribution.py#L85

### CODE REVIEW

**Comment by afuetterer** [comment]
I don't know why the CI failed. Could you trigger a re-run? I did not touch the code itself, only docs. Strange.

File: bertopic/_bertopic.py [hunk 1]
```diff
@@ -572,7 +572,7 @@ def partial_fit(self,
 
         For each subset of the data:
 
-        1. Generate embeddings with a pre-traing language model
+        1. Generate embeddings with a pre-training language model
```
Comment (MaartenGr) [inline bertopic/_bertopic.py:575]:
```suggestion
        1. Generate embeddings with a pre-trained language model
```

File: docs/getting_started/guided/guided.svg
Comment (MaartenGr) [inline docs/getting_started/guided/guided.svg:1]: What was the change here?

**Review by MaartenGr — COMMENTED** [review]
Thanks for fixing these typos! I'll have to think about the package you refer since I rather manually check whether the changes are made correctly.

File: docs/getting_started/guided/guided.svg
Comment (afuetterer) [reply docs/getting_started/guided/guided.svg:1]: There was "Mutiply the IDF values of the seeded", missing l in multiply.

**Comment by afuetterer** [comment]
Of course, you don't need to use typos, I just wanted to let you know about the option.

I accepted your suggestion. Please feel free to squash the commits.

**Comment by MaartenGr** [comment]
Awesome, thanks for the work on this!

**Pull request merged**